extensions/HyperVExtension/src/HyperVExtension/Models/VirtualMachineCreation/VMGalleryVMCreationOperation.cs

```diff
@@ -85,67 +89,82 @@ public void Report(IOperationReport value)
     /// Starts the VM gallery operation.
     /// </summary>
     /// <returns>A result that contains information on whether the operation succeeded or failed</returns>
-    public IAsyncOperation<CreateComputeSystemResult?> StartAsync()
+    public IAsyncOperation<CreateComputeSystemResult> StartAsync()
     {
         return Task.Run(async () =>
         {
-            try
+            // We'll attempt to create the VM from the gallery twice before completely failing and returning
+            // an error back to Dev Home.
+            var creationAttempt = 1;
+            var maxAttempts = 2;
```

Comment:
NIT: make the type a uint to stop negative numbers.

Reply:
updated thanks